src/theme/components/toggle.ts

```diff
@@ -0,0 +1,63 @@
+import { helperTextMargin } from "./global";
+
+// Style object for the Toggle's label
+const baseStyleLabel = {
+  fontSize: -1,
+  marginBottom: "0",
+  marginLeft: "xs",
+};
+
+// Style object for the Toggle's helper text
+const baseStyleHelper = {
+  ...helperTextMargin,
+  marginLeft: "xs",
+};
+
+const baseStyle = {
+  alignItems: "center",
+  label: baseStyleLabel,
+  helper: baseStyleHelper,
+};
+
+const Switch = {
+  baseStyle: {
+    opacity: 0.4,
+    track: {
```

Comment:
To address the focus problem, add the following to `track`:
```
_focus: {
  outline: "2px solid",
  outlineColor: "ui.focus",
  outlineOffset: "2px",
  zIndex: "9999",
},
```

Similar to `activeFocus` in global.ts, it would make sense to also add this style object that we can reuse.  Maybe call it `activeFocusNoBorder`.

Cc @EdwinGuzman